pkg/odo/cli/config/set.go

```diff
@@ -81,6 +82,12 @@ func (o *SetOptions) Complete(name string, cmd *cobra.Command, args []string) (e
 		IsNow:                  o.now,
 		CheckRouteAvailability: checkRouteAvailability,
 	})
+	if err != nil {
+		if err1 := util.IsInvalidKubeConfig(err); err1 != nil {
```

Comment:
I'm sorry for going back and forth on this one, but can you rename this function to `IsInvalidKubeConfigErr` or something like that, because right now it seems that it should return a bool value like before, but it isn't.

Reply:
done